demos/browser/src/App.tsx

```diff
@@ -627,10 +627,23 @@ const Agent: React.FC<{}> = props => {
 
 
 function App() {
+  const [pluto, setPluto] = useState<SDK.Domain.Pluto>()
+
+  useEffect(() => {
+    const defaultPassword = new Uint8Array(32).fill(1);
+    Database.createEncrypted(
+      {
+        name: `my-db`,
+        encryptionKey: defaultPassword,
+        storage: InMemory,
+      }
+    ).then((db) => setPluto(db as any))
+  })
```

Comment:
pretty sure this is an infinite loop, needs dependency to prevent triggering:
```suggestion
  useEffect(() => {
    const defaultPassword = new Uint8Array(32).fill(1);
    Database.createEncrypted(
      {
        name: `my-db`,
        encryptionKey: defaultPassword,
        storage: InMemory,
      }
    ).then((db) => setPluto(db as any))
  },  [])
```

Reply:
Applied something even better, on both next and react projects.

```
  useEffect(() => {
    if (!pluto) {
      const defaultPassword = new Uint8Array(32).fill(1);
      Database.createEncrypted(
        {
          name: `my-db`,
          encryptionKey: defaultPassword,
          storage: InMemory,
        }
      ).then((db) => setPluto(db as any))
    }
  }, [pluto, setPluto])
```

Thanks for the suggestion!